android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java

```diff
@@ -226,8 +226,7 @@ protected void onPostExecute(Integer result) {
     }
 
     /**
-     * Download a non-KMP Keyman keyboard.
-     * This will either be from Keyman cloud or legacy ad-hoc distribution via JSON
+     * Download a non-KMP Keyman keyboard from Keyman cloud via JSON
```

Comment:
We'll be removing code for pulling non-KMP downloads from Keyman cloud as well soon -- i.e. downloading all files by KMP from Keyman cloud as well. But can put that in a separate PR once we are sure the backend is ready for it.

Reply:
👍 btw, the diff in this file is cleaner when ignoring whitespace
```
https://github.com/keymanapp/keyman/pull/1109/files?w=1
```